src/components/external-source/ExternalSourceManager.svelte

```diff
@@ -0,0 +1,1050 @@
+      <svelte:fragment slot="header">
+        <slot name="left">
+          <SectionTitle><Truck slot="icon" />External Sources</SectionTitle>
+          <div class="filter">
+            <div class="timeline-editor-layer-filter">
+              <Input>
+                <input
+                  type="search"
+                  bind:value={filterExpression}
+                  placeholder="Filter External Sources"
+                  class="st-input"
+                />
+              </Input>
+            </div>
+          </div>
+        </slot>
+        <slot name="right">
+          <button
+            name="create-groups-or-types"
+            class="st-button secondary"
+            on:click|stopPropagation={onCreateGroupsOrTypes}
+            use:tooltip={{
+              content: 'Create derivation groups, external source types, or external event types.',
+              placement: 'top',
+            }}
+          >
+            Create New Groups or Types
+          </button>
+          <button
+            name="manage-groups-or-types"
+            class="st-button active"
+            on:click|stopPropagation={onManageGroupsAndTypes}
+            use:tooltip={{
+              content:
+                'Manage and inspect existing derivation groups, external source types, and external event types.',
+              placement: 'top',
+            }}
+          >
+            Manage Groups and Types
+          </button>
+        </slot>
+      </svelte:fragment>
+      <svelte:fragment slot="body">
+        {#if $externalSources.length}
+          <BulkActionDataGrid
+            {columnDefs}
+            {hasDeletePermission}
+            singleItemDisplayText="External Source"
+            pluralItemDisplayText="External Source"
+            {filterExpression}
+            items={filteredExternalSources}
+            {user}
+            getRowId={getRowIdExternalSourceSlim}
+            on:rowClicked={({ detail }) => selectSource(detail.data)}
+            on:bulkDeleteItems={({ detail }) => onDeleteExternalSource(detail)}
+            bind:selectedItemId={selectedSourceId}
+          />
+        {:else}
+          <p>No external sources matching the selected external source type(s).</p>
+        {/if}
+      </svelte:fragment>
+    </Panel>
+
+    {#if selectedSource}
+      <CssGridGutter track={1} type="row" />
+
+      <!-- External Event Table/Timeline -->
+      <Panel padBody={true}>
+        <svelte:fragment slot="header">
+          <slot name="left">
+            <SectionTitle><Balloon slot="icon" />External Events</SectionTitle>
+            <div class="filter">
+              <div class="timeline-editor-layer-filter">
+                <Input>
+                  <input
+                    bind:value={externalEventsTableFilterString}
+                    autocomplete="off"
+                    class="st-input w-100"
+                    name="filter-ee"
+                    placeholder={'Filter external events'}
```

Comment:
Prefer `placeholder='Filter external events'` here and for all other props where the assignment is just a string

Reply:
If you search for `={'` you'll find all the other instances though not all of them are from your PR so you could skip those.

Reply:
Addressed in [remove extraneous instances of ={'](https://github.com/NASA-AMMOS/aerie-ui/pull/1396/commits/a1094cff59f3cf07045da41e324e741074a41eeb)